index.html

```diff
@@ -30,6 +30,44 @@ <h2>What I do</h2>
       <p>
         <a href="https://www.linkedin.com/in/olysu/">view full cv</a>
       </p>
+      <!--work section-->
+      <h2>work I've done</h2>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 work-1">
```

Comment:
1. For the grid, please consider responsive design so we're stacking on small screens and then 2 columns from small screens and up (see Bootstrap's [column breaks]
(https://getbootstrap.com/docs/5.0/layout/columns/#column-breaks))
2. We could consider Bootstrap's [card component](https://getbootstrap.com/docs/5.0/components/card/)
3. We should also consider Bootstrap's [card group](https://getbootstrap.com/docs/5.0/components/card/#card-groups)
4. I don't think we need the `work-1` CSS class name if it's not used for styling. If we want to convey number ordering then we should use a different approach